### DISCUSSION TITLE
FT232HL Circuit Design in progress

### DISCUSSION THREAD

**Discussion by arieslaptop** [nofeletru/UsbAsp-flash#113] [Ideas]
FT232HL BASED 
![ft232h](https://github.com/nofeletru/UsbAsp-flash/assets/88767405/7001dd26-1eb4-4196-a6b6-1abb73899142)

**Comment by JhoushuTomato** [comment]
私は、今、FTDI社のFT232Hを利用して、RenesasのRXシリーズやSH2Aをデバッグしたいと考えてます。
あなたのこの設計中の回路は、この用途に利用できますか？　Jhoushu_tomato

**Reply by arieslaptop** [reply]
hi, is that jtag debug connection? if yes you have bsdl file of debugging?

**Comment by JhoushuTomato** [comment]
差出人: Harshad Patel ***@***.***>
送信日時: 2024年4月3日 22:13
宛先: nofeletru/UsbAsp-flash ***@***.***>
CC: Jhoushu-Tomato ***@***.***>; Comment ***@***.***>
件名: Re: [nofeletru/UsbAsp-flash] FT232HL Circuit Design in progress (Discussion #113)


hi, is that jtag debug connection? if yes you have bsdl file of debugging?

—
Reply to this email directly, view it on GitHub<https://github.com/nofeletru/UsbAsp-flash/discussions/113#discussioncomment-8996967>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AGS462AZSOWIOJI4DFS7NZ3Y3P56LAVCNFSM6AAAAABDHZ2HFKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSOJWHE3DO>.
You are receiving this because you commented.Message ID: ***@***.***>
Sorry for asking you a question when you were busy.
I'm not familiar with debugging tools at all.
However, I recently came into contact with the Renesas-RX series, and the development environment released by the manufacturer is very expensive.
I've been searching various websites to try and create a development environment like Eclipse, connect it to USB via JTAG, and create a tool like OpenOCD.
So, I don't have anything ready yet. I'm currently studying JTAG and MESSE. Please give me some guidance if you like. Thank you.

**Reply by arieslaptop** [reply]
Basically, this circuit design was for spi flash programming, and because the asprogrammer software for the programmer supported the ft232h chipset, I was given the opportunity to design hardware for the programmer. Sincere gratitude is extended to **Mr. Alexander Nofeletru** for the development of the asprogrammer software for the spi flash.
 

The FT232H series makes it simple to convert USB to jtag. The royalty-free Virtual Com Port (VCP) and Direct (D2XX) drivers offered by FTDI eliminate the need for USB driver development in the majority of situations. You can use this programmer to convert USB to jtag out. Additionally, this device comes equipped with an eProm that can be configured according to the protocol that you require with the assistance of ft-prog (for additional information regarding the ft232h feature, please refer to the link provided). ftdichip.com/products/ft232hq/ 


![20240404_123529](https://github.com/nofeletru/UsbAsp-flash/assets/88767405/0723e59f-f06d-4428-ad17-ab5d785ec8c0)

![20240404_123615](https://github.com/nofeletru/UsbAsp-flash/assets/88767405/526af73f-18f8-4b85-8896-5d6bb57e303c)

**Comment by JhoushuTomato** [comment]
Thank you very much for all the information.
By the way, are the circuit diagrams and board design drawings of the FT232H board that you designed publicly available?
Also, if you are manufacturing printed circuit boards, is it possible for you to share the circuit boards for a fee?

I would like to make one set as a USB->JTAG interface and the other as a normal USB->RS232C converter.

I looked through the web reference page you gave me, and I thought application notes AN_129, AN_135, and AN_108 would be helpful, so I'm currently frantically reading them.
You will also learn about the FTDI driver software.

I'm not good at English, so I machine-translated this sentence while checking it with Google Translate.
Please forgive me if the text is incorrect.
Thank you for spending your precious time with us. ．....ありがとうございます。
________________________________
差出人: Harshad Patel ***@***.***>
送信日r: 2024年4月4日 16:11
宛先: nofeletru/UsbAsp-flash ***@***.***>
CC: Jhoushu-Tomato ***@***.***>; Comment ***@***.***>
件名: Re: [nofeletru/UsbAsp-flash] FT232HL Circuit Design in progress (Discussion #113)


The FT232H series makes it simple to convert USB to jtag. The royalty-free Virtual Com Port (VCP) and Direct (D2XX) drivers offered by FTDI eliminate the need for USB driver development in the majority of situations. You can use this programmer to convert USB to jtag out. Additionally, this device comes equipped with an eProm that can be configured according to the protocol that you require with the assistance of ft-prog (for additional information regarding the ft232h feature, please refer to the link provided). ftdichip.com/products/ft232hq/

20240404_123529.jpg (view on web)<https://github.com/nofeletru/UsbAsp-flash/assets/88767405/0723e59f-f06d-4428-ad17-ab5d785ec8c0>

20240404_123615.jpg (view on web)<https://github.com/nofeletru/UsbAsp-flash/assets/88767405/526af73f-18f8-4b85-8896-5d6bb57e303c>

―
